runtime/gc_base/GCExtensions.hpp

```diff
@@ -197,6 +197,7 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 	UDATA freeSizeThresholdForSurvivor; /**< if average freeSize(freeSize/freeCount) of the region is smaller than the Threshold, the region would not be reused by collector as survivor, for balanced GC only */
 	bool recycleRemainders; /**< true if need to recycle TLHRemainders at the end of PGC, for balanced GC only */
 
+	bool disableConcurrentSupportForContinuation;
```

Comment:
Would you please add annotation to this variable?

Reply:
I actually don't want this debug flag into official builds. Please remove it from this PR, and perform all the testing relevant to it in local test environment.